core/templates/domain/statistics/playthrough-issue.model.ts

```diff
@@ -114,17 +111,14 @@ export class CyclicStateTransitionsPlaythroughIssue extends
   }
 }
 
-export type PlaythroughIssue = (
+export type PlaythroughIssueType = (
```

Comment:
I think it's incorrect to call this a type because it doesn't describe a type (unlike e.g. ExplorationTaskType). Can we just get rid of this type export and use PlaythroughIssue to represent a variable that has that type?

Reply:
For now, I don't see a way to get rid of this export. In the code used abstract class for creating different issues thus we need to know all created issues classes.  

So I restored the old name `PlaythroughIssue` and changed the name for the model class to `PlaythroughIssueModel` (like in exploration-task.model.ts)  

While I tried to understand code I replaced all hardcoded issue names with enumerator. Is it welcome?

Reply:
Thanks @DmirtyUsov, sorry for the late reply. Using an enum is definitely welcome, thank you for fixing that!

I think your approach is probably fine but I wanted to ask if we could avoid having two names mean the same thing (i.e. IssueType and PlaythroughIssueType)? I think it would be better if we just called everything PlaythroughIssueType to avoid any confusion, and got rid of IssueType entirely. Would that be possible?

Thanks!

Reply:
@seanlip I replaced `IssueType` with `PlaythroughIssueType`. It seems good.
Next, we need to refactor `PlaythroughObjectFactory`. I hope during this process we can reduce `export`s.